.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,32 @@
+name: Run CodeQL
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '24 2 * * 4'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'csharp' ]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v1
+        with:
+          languages: ${{ matrix.language }}
+          queries: +security-and-quality
```

Comment:
I'm assuming this will do linting.
In which case, adding this to all the plugins will likely be a fair bit of work, as many of them haven't gotten a proper linting pass since before our time.

Reply:
Well this is CodeQL, it is first and foremost set up to find and identify obvious security problems within code, yet allows for the analysis of quality too. I have not seen any of those quality results myself yet unfortunately, so I cannot 100% say if it will act as a linter per say. However this is already live in [jellyfin-vue](https://github.com/jellyfin/jellyfin-vue/blob/master/.github/workflows/codeql-analysis.yml), where I got the inspiration to use this query from initially.

Reply:
Was just a heads up for a potential problem for when adding to the rest of the plugins :smile: